src/portable/st/synopsys/dcd_synopsys.c

```diff
@@ -416,7 +416,9 @@ void dcd_init (uint8_t rhport)
   if ( rhport == 1 )
   {
     // On selected MCUs HS port1 can be used with external PHY via ULPI interface
-
+#if defined(TUD_OPT_SYNOPSYS_FS_PHY)
```

Comment:
you should use the builtin highspeed test instead of introducing new macro, if `CFG_TUSB_RHPORT1_MODE` is not OPT_MODE_HIGH_SPEED, we can use the fullspeed phy  

```
#if TUD_OPT_HIGH_SPEED
```